README.md

```diff
@@ -134,7 +157,7 @@ Do you need help? Do you want to chat? [Join our Gitter channel](https://gitter.
 
 _\* databases are optional, however bot memory will not persist between different connectors or system reboots without one_
 
-## Contributors
+### Contributors
```

Comment:
Why have you chosen to indent this further? This makes it appear to be a subtitle of "Usage", which it isn't.

Reply:
Ignore this, GitHub had collapsed things and I got confused 🙃